corehq/apps/hqcase/management/commands/reindex_es_native.py

```diff
@@ -117,16 +117,6 @@ def _initialize_target(es, target_index_info):
 
 
 def start_reindex(es, source_index, target_index):
-    reindex_query = {
-        "source": {
-            "index": source_index,
-        },
-        "dest": {
-            "index": target_index
-        },
-        "conflicts": "proceed"
-    }
-
     result = manager.reindex(source_index, dest_index, wait_for_completion=False)
     return task_id
```

Comment:
Huh, do you know what's up with this warning?

Reply:
yep, i used the wrong variable name. fixed it and force pushed to clean up the history a bit